src/mat3x4.rs

```diff
@@ -0,0 +1,984 @@
+use crate::{Mat4, Quat, Vec3, Vec4};
+
+#[cfg(not(feature = "std"))]
+use num_traits::Float;
+
+#[cfg(all(
+    target_feature = "sse2",
+    not(feature = "scalar-math"),
+    target_arch = "x86"
+))]
+use core::arch::x86::*;
+#[cfg(all(
+    target_feature = "sse2",
+    not(feature = "scalar-math"),
+    target_arch = "x86_64"
+))]
+use core::arch::x86_64::*;
+
+#[cfg(all(target_feature = "sse2", not(feature = "scalar-math")))]
+#[inline(always)]
+unsafe fn m128_from_vec4(vec4: Vec4) -> __m128 {
```

Comment:
There's already a `From` impl to do this, you can use `into()`.

Reply:
Oh :)